### ISSUE TITLE
Add to dashboard buttons

### ISSUE THREAD

**Issue by matyax** [grafana/explore-logs#861]
Similar to Go to Explore, we need to consider an option to add to a dashboard, and also to add breakdown panels to dashboards as well.

**Comment by zizzpudding** [comment]
I think we need to consider having a menu on the breakdown panel cards as the number of actions is growing, leaving less room for title. 
Either a menu option or a better way of showing buttons. 
For instance, adding a button to add to dashboard and add to investigation is not going to scale in this case. 


![Image](https://github.com/user-attachments/assets/1fb05d4f-ad4b-4e4a-99bf-d789ebad989c)

**Comment by gtk-grafana** [comment]
This is possible using the [API](https://grafana.com/docs/grafana/latest/developers/http_api/dashboard/#create--update-dashboard), but we'd have to do a lot of wheel-reinventing to let folks add a panel to an existing dashboard. We could add a panel to a new dashboard, but I'm not sure that would be useful on its own.

For now users can workaround by adding a panel to explore, and then using the add to dashboard functionality there.

**Comment by ifrost** [comment]
Related issue: https://github.com/grafana/grafana/issues/90834

It would be nice if it was an extension point exposed from dashboards so any app can use it easily. (cc @grafana/dashboards-squad )

**Comment by gtk-grafana** [comment]
Yeah, see the "For Follow-up PRs" section of https://github.com/grafana/grafana/pull/94458, either a core scenes method or extension point would be wonderful

**Comment by ivanortegaalba** [comment]
Hi team! 

Sure, we will be happy to accept suggestions, DD, and PRs! Feel free to send them our way, and we will happily help to get it done 👍
